docs/contivctl.md

```diff
@@ -0,0 +1,227 @@
+## contivctl
+
+contivctl command line tool lets you add/modify/delete objects into contiv object model using REST api.
+
+*Note:*
+`contivctl` Command can be found under `$GOPATH/bin` directory (`/opt/gopath/bin` in vagrant VMs) after a build. 
+Currently, `contivctl` needs to be run on the same host where `netmaster` is running
+
+## Usage
+
```

Comment:
Would you think it makes sense to provide with 3 steps for someone to try out a policy between two containers?
I think it makes perfect sense to keep the usage, etc. but before any of that, we can let users try as they understand the concepts.